charts/workflow/values.yaml

```diff
@@ -51,6 +51,16 @@ global:
   host_port: 5555
   # Prefix for the imagepull secret created when using private registry
   secret_prefix: "private-registry"
+  # The public resolvable hostname to build your cluster with.
+  #
+  # This will be the hostname that is used to build endpoints such as "deis.$HOSTNAME"
+  hostname: ""
```

Comment:
@mboersma As it stands we will get a regex error **before** we install the chart.

```
kris-nova:workflow kris$ helm install . --namespace deis --set global.experimental_native_ingress=true
Error: release elegant-cardinal failed: Ingress.extensions "controller-api-server-ingress" is invalid: spec.rules[0].host: Invalid value: "deis.": must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')
kris-nova:workflow kris$ 
```